rustls/src/client/tls12.rs

```diff
@@ -31,7 +31,7 @@ use crate::client::common::ClientAuthDetails;
 use crate::client::common::ServerCertDetails;
 use crate::client::{hs, ClientConfig, ServerName};
 
-use ring::constant_time;
+use subtle;
```

Comment:
I'm on board with this, but for this PR I feel like we shouldn't keep the commit + revert in there.